### PR TITLE
Drop apollo-server as dependency

### PR DESCRIPTION
**Pull request by cabelitos**
This will help people using apollo-server-lambda, apollo-server-express
to do not have to add the apollo-server package as dependency
to use this package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,13 +24,14 @@
     "yarn": "^1.21.1"
   },
   "dependencies": {
+    "graphql-tag": "^2.11.0",
+    "graphql-tools": "^4.0.0",
     "lodash.difference": "^4.5.0",
-    "yargs": "^15.4.1"
+    "yargs": "^16.0.3"
```
Comment (barbieri) [inline package.json:30]: yargs should be devDependency, no?

**Review by barbieri — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (cabelitos) [reply package.json:30]: @barbieri The script used to generatype the typeMaps json uses yargs. This is why I left in dependencies.

Comment (barbieri) [reply package.json:30]: maybe peer?

Comment (cabelitos) [reply package.json:30]: Not sure, at least the packages that I looked some the same strategy. Take a look for example in typeorm https://github.com/typeorm/typeorm/blob/master/package.json#L132
I think we should leave it as dependency
@barbieri

Comment (barbieri) [reply package.json:30]: ok, then

**Comment by barbieri** [comment]
lgtm, merge

**Pull request merged**